apps/cli-daemon/src/app/generators/utils.ts

```diff
@@ -0,0 +1,49 @@
+import { readdirSync, readFileSync } from 'fs';
+import { join } from 'path';
+
+import { GeneratorDefinition, Schema } from './generators.interface';
+
+const capitalize = (s: string): string => s[0].toUpperCase() + s.slice(1);
+const formatSchemaName = (name: string): string => {
+  const splitName = name.split('-');
+  if (splitName.length > 1) {
```

Comment:
you can use .map on the splitName and capitalize every entry, it will work also for more than 2 words 
or is it just for 2 by purpose?

Reply:
have you seen this comment?

Reply:
updated